src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -379,6 +378,27 @@ namespace OpenMS
     return false;
   }
 
+  String MRMDecoy::getModifiedPeptideSequence_(const OpenMS::TargetedExperiment::Peptide& pep) const
+  {
+    String full_peptide_name;
+    for (int loc = -1; loc <= (int)pep.sequence.size(); loc++)
+    {
+      if (loc > -1 && loc < (int)pep.sequence.size())
+      {
+        full_peptide_name += pep.sequence[loc];
+      }
+      // C-terminal and N-terminal modifications may be at positions -1 or pep.sequence
+      for (Size modloc = 0; modloc < pep.mods.size(); modloc++)
+      {
+        if (pep.mods[modloc].location == loc)
+        {
+  	full_peptide_name += "(UniMod:" + String(pep.mods[modloc].unimod_id) + ")";
```

Comment:
there are still some style errors. e.g. here no indentation. below it uses probably tabs instead of two spaces. maybe give it another quick check?

Reply:
I will give try to give a quick fix to move this forward

Reply:
can't push to Roestlab... I think you need to do the changes yourself. sorry

Reply:
no problem I'll take a look